integration-tests/src/test/resources/features/end2endtestit.feature

```diff
@@ -107,3 +107,75 @@ Feature: End to End Bridge integration tests
       ]
     }
     """
+
+  Scenario: Slack Action Processor is created and slack message should be received in the slack channel
+    Given authenticate against Manager
+
+    When create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    Then add a Processor to the Bridge "myBridge" with body:
+    """
+    {
+      "name": "slackProcessor",
+      "action": {
+        "type": "Slack",
+        "parameters": {
+            "channel": "mc",
+            "webhookUrl": "https://hooks.slack.com/services/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"
+            }
+      },
+      "filters": [
+        {
+        "key": "source",
+        "type": "StringEquals",
+        "value": "StorageService"
+        }
+       ],
+  	  "transformationTemplate": "{data.myMessage}"
+    }
+    """
+    And the Processor "slackProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+    And the Processor "slackProcessor" of the Bridge "mybridge" has action of type "Slack"
+
+    Then add a Processor to the Bridge "mybridge" and returns HTTP response code 400 with body:
+    """
+    {
+      "name": "processorInvalid",
+      "action": {
+        "type": "SlackAction",
+        "properties": {
+            "channel": "mc",
+            "webhookUrl": "https://hooks.slack.com/services/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"
+      }
+      },
+       "filters": [
+        {
+        "key": "source",
+        "type": "StringEquals",
+        "value": "StorageService"
+        }
+      ]
+    }
+    """
+
+    And send a slack message cloud event to the Ingress of the Bridge "mybridge":
```

Comment:
May be better to utilize cloud event id from https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/409 (once it gets merged), that way you can use default cloud event step.
In the verification step you can then check that id is received in the Slack.